spec/presenters/jobs_presenter.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper'
```

Comment:
@notapatch I noticed there's a `spec/presenters/job_presenter_spec.rb` file and a `spec/presenters/jobs_presenter.rb` file, do we need both?

Reply:
Interesting. So, job_presenter is a singular presenter and job**s**_presenter is a collection of presenters. I would say that the collection tests were named incorrectly. 

| Model | Tests |
|-----|---------|
| `app/presenters/job_presenter.rb`  | `spec/presenters/job_presenters_spec.rb` |
| `app/presenters/jobs_presenter.rb` | ~spec/presenters/jobs_presenters.rb~  |

It should have been named **jobs_presenter_spec.rb** . I will make that change. 

I am not sure why this is the only collection presenter and it might be worth knowing why that is. I don't know enough about jobs_presenter other than it is used in the app.

Reply:
@notapatch makes sense, thanks for updating the name of the spec.